pom.xml

```diff
@@ -40,11 +40,11 @@
         <!-- NOTE: Jetty needed for Solr, Handle Server & tests -->
         <jetty.version>9.4.57.v20241219</jetty.version>
         <log4j.version>2.24.3</log4j.version>
-        <pdfbox-version>2.0.34</pdfbox-version>
+        <pdfbox-version>3.0.5</pdfbox-version>
         <rome.version>1.19.0</rome.version>
         <!-- SLF4J is not used, but specified for dependency convergence issues between many dependencies -->
         <slf4j.version>2.0.17</slf4j.version>
-        <tika.version>2.9.4</tika.version>
+        <tika.version>3.1.0</tika.version>
         <!-- Sync BouncyCastle & ASM with whatever version Tika uses -->
         <bouncycastle.version>1.80</bouncycastle.version>
         <asm.version>8.0.1</asm.version>
```

Comment:
We should also be pinning the asm dependency in `pom.xml` to 9.7.1 to match that of Tika 3.1.0: https://github.com/apache/tika/blob/3.1.0/tika-parent/pom.xml#L318